src/Symfony/Component/Console/Tests/phpt/single_application/version_name.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Single Application can be executed
+--ARGS--
+--version
+--FILE--
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SingleCommandApplication;
+
+$vendor = __DIR__;
+while (!file_exists($vendor.'/vendor')) {
+    $vendor = dirname($vendor);
+}
+require $vendor.'/vendor/autoload.php';
```

Comment:
Why not use `require dirname(__DIR__, 3).'/vendor/autoload.php` instead of the while loop which is infinite if the vendor directory does not exist?

Reply:
I copied the this from other phpt tests. 
I guess it is to manage per components vs global tests suite